docusaurus/docs/making-a-progressive-web-app.md

```diff
@@ -34,14 +40,35 @@ However, they [can make debugging deployments more challenging](https://github.c
 
 The [`workbox-webpack-plugin`](https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin)
 is integrated into production configuration,
-and it will take care of generating a service worker file that will automatically
+and it will take care of compiling a service worker file that will automatically
 precache all of your local assets and keep them up to date as you deploy updates.
 The service worker will use a [cache-first strategy](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)
 for handling all requests for local assets, including
 [navigation requests](https://developers.google.com/web/fundamentals/primers/service-workers/high-performance-loading#first_what_are_navigation_requests)
 for your HTML, ensuring that your web app is consistently fast, even on a slow
 or unreliable network.
 
+## Customization
+
+Starting with Create React App 3, you have full control over customizing the
```

Comment:
There's a couple of references to `c-r-a` 3 here. I'm not sure if this PR will actually be merged in to `c-r-a` 3, though, so this might need to be changed.

Reply:
I've updated to `4`.